src/plugins/auto_batch/src/sync_infer_request.cpp

```diff
@@ -338,12 +114,24 @@ void SyncInferRequest::CopyBlobIfNeeded(InferenceEngine::Blob::CPtr src, Inferen
     }
 }
 
-void SyncInferRequest::CopyOutputsIfNeeded() {
-    for (const auto& it : _networkOutputs) {
-        auto& name = it.first;
+void SyncInferRequest::copy_outputs_if_needed() {
+    for (const auto& it : get_outputs()) {
         // this request is already in BUSY state, so using the internal functions safely
-        CopyBlobIfNeeded(m_batched_request_wrapper._inferRequestBatched->GetBlob(name), GetBlob(name), false);
+        auto dst_tensor = get_tensor(it);
+        copy_tensor_if_needed(m_batched_request_wrapper->_infer_request_batched->get_tensor(it), dst_tensor, false);
     }
 }
+
+void SyncInferRequest::infer() {
+    OPENVINO_NOT_IMPLEMENTED;
+}
+
+std::vector<ov::SoPtr<ov::IVariableState>> SyncInferRequest::query_state() const {
+    return m_batched_request_wrapper->_infer_request_batched->query_state();
```

Comment:
You need to wrap so in some cases, please take a look to hetero or proxy plugins